TicketMate.Persistence/DataRequestObjects/TicketRequests/UpdateTicketByGuid.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace TicketMate.Persistence.DataRequestObjects.TicketRequests
+{
+    /// <summary>
+    /// Todo...
+    /// </summary>
+    public class UpdateTicketByGuid : IDataExecute
+    {
+        public object? GetParameters() => this;
+        public string GetSql() => "";
```

Comment:
Are we intentionally not doing anything with this request?

Reply:
Yes, That will be apart of the next PR.